tests/e2e/e2e_test.go

```diff
@@ -126,8 +126,8 @@ func (s *IntegrationTestSuite) TestSuperfluidVoting() {
 	)
 }
 
-// TestAddToExistingLockPostUpgrade ensures addToExistingLock works for locks created preupgrade.
-func (s *IntegrationTestSuite) TestAddToExistingLockPostUpgrade() {
+// Test03AddToExistingLockPostUpgrade ensures addToExistingLock works for locks created preupgrade.
+func (s *IntegrationTestSuite) Test03AddToExistingLockPostUpgrade() {
```

Comment:
Same with this one, it shouldn't need a number appended to it

Reply:
Same reason here